stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -246,16 +250,20 @@ spec:
             httpGet:
               path: /
               port: www
-            initialDelaySeconds: 10
-            failureThreshold: 6
-            timeoutSeconds: 5
+            initialDelaySeconds: {{ default 10 .Values.pipelines.www.livenessProbe.initialDelaySeconds }}
+            timeoutSeconds: {{ default 5 .Values.pipelines.www.livenessProbe.timeoutSeconds }}
+            periodSeconds: {{ default 5 .Values.pipelines.www.livenessProbe.timeoutSeconds }}
+            failureThreshold: {{ default 6 .Values.pipelines.www.livenessProbe.failureThreshold }}
+            successThreshold: {{ default 1 .Values.pipelines.www.livenessProbe.successThreshold }}
           readinessProbe:
             httpGet:
               path: /
```

Comment:
@stein321 Can we also move `path` and `port` to values.yaml

Reply:
resolved